src/sphmultipcqueue.cpp

```diff
@@ -10,7 +10,7 @@
  *     IBM Corporation, Paul Clarke   - MPMC implementation
  */
 
-#define __SASDebugPrint__ 1
+//#define __SASDebugPrint__ 1
```

Comment:
This is marked as a comment in "an outdated diff"?

I'd prefer to keep this line in, as an easy way to enable debug output.